src/App.js

```diff
@@ -1,19 +1,19 @@
-import React from 'react';
-import {BrowserRouter as Router, Switch, Route} from 'react-router-dom'
-import './App.css';
-import Header from './components/header/header';
-import Home from './components/home/Home';
-
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import "./App.css";
+import Header from "./components/header/header";
+import Home from "./components/home/Home";
+import SurveyResult from "./components/SurveyResult/SurveyResult";
```

Comment:
I guess there is a mistake in the naming convention of the files, because of which the build was failing.
Please rename your files from `surveyResult` to `SurveyResult` & `surveyResult.js` to `SurveyResult.js`

![Screenshot 2020-10-24 at 2 32 11 PM](https://user-images.githubusercontent.com/54791364/97077988-0f76f080-1606-11eb-903e-f14dd373adae.png)